ETC.tsv

```diff
@@ -97,109 +97,109 @@ ETC_20150317_000096	$Although their bright color makes them pretty visible, it's
 ETC_20150317_000097	$Yellow Griba
 ETC_20150317_000098	$Large Red Griba
 ETC_20150317_000099	$Big Griba
-ETC_20150317_000100	Someone once described this monster as hard to block and hard to eat.
+ETC_20150317_000100	$Someone once described this monster as hard to block and hard to eat.
 ETC_20150317_000101	$Big Blue Griba
-ETC_20150317_000102	A large loathsome creature that is intentionally hunted.
+ETC_20150317_000102	$Large, strong, and tough monster which is a reluctant prey of the hunters
 ETC_20150317_000103	$Green Big Griba
 ETC_20150317_000104	$Red Big Griba
 ETC_20150317_000105	$Old Kepa
-ETC_20150317_000106	Kepas have aged the Pokusaku flower through constantly leaching it. Caution is advised as it emits a strong spicy scent.
-ETC_20150317_000107	Gelsvas Old Kepa
+ETC_20150317_000106	$After putting so much effort to bloom its flower, a Kepa became so old. Be careful of its spicy scent which is stronger now.
```

Comment:
@emailboxu Thank you :) I'll update the first sentence then discuss the 2nd sentence on discord